data/Makefile-import-data.jinja2

```diff
@@ -17,7 +17,7 @@ import-shapefiles.sh: {{ tgt_shapefile_shps }}
 shapefiles: shapefiles.tar.gz
 
 shapefiles.tar.gz:
-	wget 'http://s3.amazonaws.com/{{ bucket }}/{{ datestamp }}/shapefiles.tar.gz'
+	wget --quiet 'http://s3.amazonaws.com/{{ bucket }}/{{ datestamp }}/shapefiles.tar.gz'
```

Comment:
Add print out to say we're doing **something**, and add comment saying the `--quiet` is to be friendlier with Vagrant.